controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go

```diff
@@ -108,6 +108,8 @@ type KubeadmControlPlaneStatus struct {
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
 // +kubebuilder:subresource:scale:specpath=.spec.replicas,statuspath=.status.replicas,selectorpath=.status.selector
+// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=".status.ready",description="Total number of clusters ready"
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="Ready",type=boolean,JSONPath=".status.ready",description="KubeadmControlPlane API Server is ready to receive requests."
```

I pulled the comment out from the GoDoc of the variable. 
https://github.com/kubernetes-sigs/cluster-api/blob/54420c41251541fdc14ed2ebd59891d540085afd/controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go#L84-L92